entrypoint.sh

```diff
@@ -37,9 +37,9 @@ echo "Git email: $(git config --get user.email)"
 echo "Running cz: $INPUT_DRY_RUN $INPUT_COMMIT $INPUT_CHANGELOG $INPUT_PRERELEASE"
 
 if [ $INPUT_CHANGELOG_INCREMENT_FILENAME ]; then
-  cz bump --yes --changelog-to-stdout $INPUT_COMMIT $INPUT_DRY_RUN $INPUT_CHANGELOG $INPUT_PRERELEASE >$INPUT_CHANGELOG_INCREMENT_FILENAME
+  cz $INPUT_NO_RAISE bump --yes --changelog-to-stdout $INPUT_COMMIT $INPUT_DRY_RUN $INPUT_CHANGELOG $INPUT_PRERELEASE >$INPUT_CHANGELOG_INCREMENT_FILENAME
 else
-  cz bump --yes $INPUT_DRY_RUN $INPUT_COMMIT $INPUT_CHANGELOG $INPUT_PRERELEASE
+  cz $INPUT_NO_RAISE bump --yes $INPUT_DRY_RUN $INPUT_COMMIT $INPUT_CHANGELOG $INPUT_PRERELEASE
```

Comment:
I don't think it's wrong, but it feels odd stylistically to pass `--no-raise` before `bump` rather than tacking it on to the list of variables at the end (maybe even in the same order they are above for consistency, but reordering them is a pre-existing choice).